include/mbedtls/threading.h

```diff
@@ -24,6 +24,12 @@
 #ifndef MBEDTLS_THREADING_H
 #define MBEDTLS_THREADING_H
 
+/*
+ * Ensure gmtime_r is available even with -std=c99; must be included before
+ * config.h, which pulls in glibc's features.h. Harmless on other platforms.
+ */
+#define _POSIX_C_SOURCE 200112L
```

Comment:
I get a compilation warning that `_POSIX_SOURCE` is being redefined here.

Reply:
I just pushed a change to fix this issue.